### PR TITLE
added margin to buttons & horizontal scroll fix

### PR DESCRIPTION
**Pull request by Onurrr**
- The buttons now have margin on them which looks better on mobile:
image
- Disabled horizontal scroll

### CODE REVIEW

File: src/components/main.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+body{
```
Comment (deathbeam) [inline src/components/main.css]: Use inline styles and for body global stylesheet in index.html

Comment (Onurrr) [reply src/components/main.css]: @deathbeam done

Comment (deathbeam) [reply src/components/main.css]: Remove this file then

**Comment by Onurrr** [comment]
@deathbeam forgot about that, my bad sorry.

It's now deleted.

File: src/components/main.css (hunk 1 above)
Comment (Onurrr) [reply src/components/main.css]: done

**Pull request closed without merging**